server/config/index.js

```diff
@@ -0,0 +1,11 @@
+import dotenv from 'dotenv';
+import { Client, Pool } from 'pg';
+dotenv.config();
+
+if (process.env.NODE_ENV === 'development') {
+  module.exports = new Client({ connectionString: process.env.DATABASE_URL });
+} else if (process.env.NODE_ENV === 'testing') {
+  module.exports = new Client({ connectionString: process.env.DB_TEST });
+} else {
+  module.exports = new Client({ connectionString: process.env.DATABASE_URL });
```

Comment:
This else statement seems redundant, it's similar to the first if statement. Just have an if statement checking if the environment is testing and then set the test DB and by default make the url the `DATABASE_URL`